custom-namespaces.yaml

```
@@ -409,6 +414,11 @@ firefox_accounts:
         - channel: release
           table: mozdata.firefox_accounts.fxa_log_device_command_events
   explores:
+    fxa_all_events:
```

The existing convention for these explore names under "Firefox Acounts" is to not have an `fxa_` prefix, have `event` singular, to have a `_counts` suffix.  So if we want to continue that convention, this would be:
```suggestion
   all_event_counts:
```